apps/docs-snippets2/src/cookbook/wallet-transferring/create-transfer-2.ts

```diff
@@ -0,0 +1,36 @@
+import { bn, Provider, Wallet } from 'fuels';
+
+import { LOCAL_NETWORK_URL, WALLET_PVT_KEY, WALLET_PVT_KEY_2 } from '../../env';
+
+const provider = await Provider.create(LOCAL_NETWORK_URL);
+const sender = Wallet.fromPrivateKey(WALLET_PVT_KEY, provider);
+const destination = Wallet.fromPrivateKey(WALLET_PVT_KEY_2, provider);
+
+// #region transferring-assets-3
```

Comment:
Should this come before the imports?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3367/commits/aefcdf3b96c79de8f7f509928e00e3d2b3dd9f7c